chatbot/serializers.py

```
@@ -0,0 +1,364 @@
+    user = UserBasicSerializer(read_only=True)
+    duration = serializers.ReadOnlyField()
+    is_active = serializers.ReadOnlyField()
+    conversation_count = serializers.SerializerMethodField()
+    
+    class Meta:
+        model = ChatbotSession
+        fields = [
+            'id', 'user', 'session_type', 'status', 'context_data',
+            'started_at', 'last_activity', 'ended_at', 'expires_at',
+            'duration', 'is_active', 'conversation_count', 'metadata'
+        ]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)